### ISSUE TITLE
the recommended version of tensorflow-gpu、keras and numpy to test the code example of "Next-Frame Video Prediction with Convolutional LSTMs"

### ISSUE THREAD

**Issue by MysteryNull** [keras-team/keras-io#1626]
### Issue Type

Others

### Source

source

### Keras Version

recommended

### Custom Code

Yes

### Python version

3.8

### Current Behavior?

pip install -r requirements.txt.
But the tensorflow in the requirements.txt can not be installed.

### Standalone code to reproduce the issue or tutorial link

```shell
1
```

**Comment by sachinprasadhs** [comment]
Hi, 

I was able to run the example mentioned above without any error using python 3.10.

Here is the Gist attached for reference https://gist.github.com/sachinprasadhs/479694ef9d6f26cedcc8cdb4c8920aab

**Comment by MysteryNull** [comment]
It works, but the performance is really bad, how can I improve the performance of the network? Thank you!
![1](https://github.com/keras-team/keras-io/assets/91356383/93a69166-e903-4136-baeb-1daf1ef5f0e3)

**Comment by sachinprasadhs** [comment]
As I can see in the above attached Gist is having very less loss.
However, the main intention of the example is to demonstrate the workflow, not to meet any benchmark performance.

In order to get better accuracy, try to increase the number of epochs and you can do some modification to the model structure if needed.

**Comment by MysteryNull** [comment]
Thank for your advice.

**Comment by sachinprasadhs** [comment]
If you don't have other question, could you please close this issue. Thanks!
